### DISCUSSION TITLE
PNG file size

### DISCUSSION THREAD

**Discussion by mohammad-torabi-84** [moononournation/Arduino_GFX#333] [Q&A] [answered]
Hi,

Is there any limitation on loading a PNG file? I have a PNG file with the dimension of 800*480 and the size of 156 KB. I am using ESP32-8048S070 as board and display. 

The file does not load on the display in one shot. So when I split it into 4 files, it loads the files but with white bars on them. Picture has been attached.

![photo_2023-08-14_16-42-28](https://github.com/moononournation/Arduino_GFX/assets/120807487/d6f2cc13-2cf3-4193-b0fa-a078867661f9)

I ended up to split it into 8 files to make it to load on the display. total loading time is 2.5 seconds

Any suggestion on how to make it working.

**Comment by moononournation** [comment]
Big PNG file is a burden for MCU, will you consider JPEG file?

**Reply by mohammad-torabi-84** [reply]
I tried the JPG example but nothing shown. I asked a question about it.

**Reply by moononournation** [reply, marked as answer]
I tried JPEG files before, it should not have problems: 【Makerfabs ESP32-S3 TFT 4.3” v1.3-哔哩哔哩】 https://b23.tv/RlW96SY
